tests/iso-8859-2-hungarian/auto-apro.hu.xml

```diff
@@ -1,474 +0,0 @@
-<?xml version="1.0" encoding="iso-8859-2" ?>
```

Comment:
Here is part of the "plain" hungarian text "honositomuhely.hu.xml' with automatically tag removed in the latin1prober:
title Honosító Műhely Legfrissebb title link http www honositomuhely hu link description description language language copyright Copyright C Herczeg József copyright pubDate Wed Jan pubDate item title Simply Calenders d fr title link http www honositomuhely hu index php option com remository Itemid func fileinfo filecatid parent link pubDate Wed Jan pubDate description description category category item item title PDF Download fw ÚJ title link http www honositomuhely hu index php option com remository Itemid func fileinfo filecatid parent link pubDate Wed Jan pubDate description description category category item item title VideoInspector fw fr title link http www honositomuhely hu index php option com remository Itemid func fileinfo filecatid parent. 
The tag filtering is completely wrong.